cache/cache.go

```diff
@@ -44,8 +45,8 @@ func SetupReleaseCache(mgr ctrl.Manager) error {
 		"spec.snapshot", releaseIndexFunc)
 }
 
-// SetupApplicationCache adds a new index field to be able to search SnapshotEnvironmentBindings by Environment.
-func SetupApplicationCache(mgr ctrl.Manager) error {
+// SetupBindingEnvironmentCache adds a new index field to be able to search SnapshotEnvironmentBindings by Environment.
+func SetupBindingEnvironmentCache(mgr ctrl.Manager) error {
 	applicationIndexFunc := func(obj client.Object) []string {
```

Comment:
Is it better to change variable name `applicationIndexFunc` to `bindingIndexFunc`? But this is not related to this function movement.

Reply:
Good idea, renamed to `bindingEnvironmentIndexFunc` for more clarity, thanks.